README.md

```diff
@@ -290,7 +290,7 @@ Models                                                                      | Ca
 [YoloNet](https://arxiv.org/abs/1506.02640)                                 |   √   |   √   |     √      |
 [Pix2Pix](https://github.com/phillipi/pix2pix)                              |   √   |   ×   |     ×      |
 [VQA](https://github.com/iamaaditya/VQA_Demo)                               |   √   |   √   |     √      |
-
+[Denoising Auto-Encoder](https://blog.keras.io/building-autoencoders-in-keras.html)                               |   ×   |   √   |     √      |
 Note: For models that use a custom LRN layer (Alexnet), Keras expects the custom layer to be passed when it is loaded from json. LRN.py is located in keras_app/custom_layers. [Alexnet import for Keras](https://github.com/Cloud-CV/Fabrik/blob/master/tutorials/keras_custom_layer_usage.md)
```

Comment:
Can you move this out of the table, I know it is not related your task but please move it out of the table.